### PR TITLE
Fixes violations of and reenables Style/UnneededInterpolation.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes generated automatically via `bundle exec rubocop --only Style/UnneededInterpolation --auto-correct`. Then manual tweaks.

### CODE REVIEW

**Comment by joshlory** [comment]
I don't have a good sense of whether the `.to_s` can be removed everywhere, but I trust that you've verified those changes.  The risk would be that these are different things:

``` ruby
irb(main):001:0> {1 => 2}
=> {1=>2}
irb(main):002:0> {1.to_s => 2}
=> {"1"=>2}
```

**Comment by joshlory** [comment]
The other risk is that `.to_s` will convert `nil` to `""`:

``` ruby
irb(main):003:0> nil.to_s
=> ""
```

**Comment by ashercodeorg** [comment]
In all cases, I verified that the variable --if not nil-- is a string. Let me go back and confirm that the variable is never nil, I missed that.

File: dashboard/app/controllers/report_abuse_controller.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def report_abuse
                 "URL: #{params[:abuse_url]}",
                 "abuse type: #{params[:abuse_type]}",
                 "user detail:",
-                "#{params[:abuse_detail]}"
+                params[:abuse_detail]
```
Comment (ashercodeorg) [inline dashboard/app/controllers/report_abuse_controller.rb:23]: Though I can't guarantee that `params[:abuse_detail]` isn't `nil`, if it is `nil`, nothing will happen (`body` won't end with a blank line).

**Comment by ashercodeorg** [comment]
Confirmed (see the one comment I added). Nice catch.

**Comment by joshlory** [comment]
LGTM.  Thanks for doing this cleanup!

**Pull request merged**